packages/ui-number-input/src/NumberInput/props.ts

```diff
@@ -166,6 +166,14 @@ type NumberInputOwnProps = {
    * sets the input type to string and allows string as value
    */
   allowStringValue?: boolean
+
+  /**
+   * Definition of the icons to be rendered for increase and decrease buttons
```

Comment:
Kinda nitpicky: Since this text acts as the documentation for the prop on the insUI docs page, I would rephrase it to something like this:

"Sets the icons to be rendered for increase and decrease buttons"

Reply:
Fixed